web-local/src/lib/types.ts

```diff
@@ -225,3 +225,11 @@ export type ColumnarTypeKeys = {
     ? K
     : never;
 }[keyof DataModelerState];
+
+/** The LayoutElement is the state needed for different layout surfaces,
+ * such as the navigation menu, the inspector, and the model output.
+ */
+export interface LayoutElement {
+  value: number;
```

Comment:
`value` is a confusing name to me. Could you name it something more descriptive? Is it `width`?

Reply:
It's unfortunately not always a width.